frameworks/FSharp/giraffe/benchmark_config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "framework": "giraffe",
+  "tests": [
+    {
+      "default": {
+        "plaintext_url": "/plaintext",
+        "json_url": "/json",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "fullstack",
+        "database": "None",
+        "framework": "giraffe",
+        "language": "F#",
+        "orm": "Raw",
+        "platform": ".NET",
+        "flavor": "CoreCLR",
+        "webserver": "Kestrel",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "Giraffe",
+        "notes": "",
+        "versus": "aspcore"
+      }
+    },
+    {
+      "jsonutf8": {
```

Comment:
Call it `utf8json` to match the library name?

Then it will appear in the listings as `giraffe-utf8json` as per https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/CSharp/aspnetcore/benchmark_config.json#L82

Url can remain the same